app/views/holidays/_form.html.erb

```diff
@@ -15,7 +15,7 @@
     </div>
   <% end %>
 
-  <% I18n.locale.to_s == 'en' ? format = 'dd-mm-yyyy' : format = 'dd.mm.yyyy' %>
+  <% I18n.locale.to_s == 'en' ? format = 'mm/dd/yyyy' : format = 'dd.mm.yyyy' %>
```

Comment:
Hier sollte Localization für das Datumsformat verwendet werden, wenn das möglich ist.
Kann dann vermutlich auch direkt in Zeile 27 und Zeile 34 eingesetzt werden.